cluster/images/conformance/run_e2e.sh

```diff
@@ -45,6 +45,10 @@ ginkgo_args=(
     "--noColor=true"
 )
 
+if [[ ${E2E_DRYRUN} ]]; then
```

Comment:
@johnSchnake would it be better to use the pattern used elsewhere?

`if [[ -n ${E2E_DRYRUN:-} ]]; then`

Reply:
https://cs.k8s.io/?q=if%20%5C%5B%5C%5B%20%5C-n.*%5C%7D%20%5C%5D%5C%5D%5C%3B%20then&i=nope&files=&repos=

Reply:
Sure, I'll make the change.